internal/providers/terraform/aws/kinesisanalytics_application.go

```diff
@@ -0,0 +1,31 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetKenesisAnalyticsApplicationRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_kinesis_analytics_application",
+		RFunc: NewKenesisAnalyticsApplication,
+		Notes: []string{
+			"Terraform doesn’t currently support Analytics Studio, but when it does they will require 2 orchestration KPUs.",
```

Comment:
This not isn't necessary on `aws_kinesis_analytics_application` since studio is a `aws_kinesisanalyticsv2_application` thing.